data_model/app/vw_sign_symbol.py

```diff
@@ -35,61 +35,86 @@ def vw_sign_symbol(srid: int, pg_service: str = None):
                 , sign.rank AS sign_rank
                 , support.id AS support_id
                 , support.geometry::geometry(Point,%(SRID)s) AS support_geometry
-                , vl_official_sign.value_de as _symbol_value_de
-                , vl_official_sign.value_fr as _symbol_value_fr
-                , vl_official_sign.value_it as _symbol_value_it
-                , vl_official_sign.value_ro as _symbol_value_ro
                 , CASE
-                  WHEN complex IS TRUE THEN 'complex.svg'
-                  WHEN fk_sign_type = 11 THEN vl_official_sign.img_de
-                  WHEN fk_sign_type = 12 THEN 'marker.svg'
-                  WHEN fk_sign_type = 13 AND mirror_red_frame IS TRUE THEN 'mirror.svg'
-                  WHEN fk_sign_type = 13 AND mirror_red_frame IS FALSE THEN 'mirror-noframe.svg'
-                  WHEN fk_sign_type = 14 THEN 'street-plate.svg'
-                END as _img_de
+                    WHEN sign.fk_sign_type = 15 THEN vl_user_sign.value_de
+                    ELSE vl_official_sign.value_de
+                  END AS _symbol_value_de
```

Comment:
If we have these in the same field, this means than the path will be dynamically set depending on the sign_type, correct?

Reply:
sorry, misread